.github/workflows/alibiexplainer_tests.yml

```diff
@@ -24,18 +24,13 @@ jobs:
   python-tests:
 
     runs-on: ubuntu-latest
+    container: seldonio/python-builder:0.6
```

Comment:
:wrench: if we could get away without using `seldonio/python-builder` image it'd be probably preferable and easier to maintain in long run

Reply:
Totally agree - but that was causing some issues (probably caused by using an old version of something somewhere). Adding back `seldonio/python-builder` was the quickest way to sort it out. 

Having said so, I'm sure that if we spent some time troubleshooting it we could get it to work with the GH standard stuff.